RichEditorView/Classes/RichEditorView.swift

```diff
@@ -439,6 +445,35 @@ extension RichEditorView: UIGestureRecognizerDelegate {
 // MARK: - Utilities
 extension RichEditorView {
     
+    /** Sets content offset for scrollView based on caret location **/
+    
+    private func fixScrollView(editor: RichEditorView) {
+        
+        var scrollView = editor.webView.scrollView
+        
+        var contentHeight: CGFloat?
+        let htmlHeight = editor.runJS("document.getElementById('editor').clientHeight;")
```

Comment:
I agree. Even in this case, since `document.getElementById('editor').clientHeight` is literally the same as `RE.editor`, I might not be too opposed to just `RE.editor.clientHeight` (I'm not sure how I'm doing it in other places.)